### PR TITLE
Introduce PEP8 code formatting

### PR DESCRIPTION
**Pull request by utdrmac**
This PR introduces a coding standard to TRD. The following steps were performed:

* **Analysis**: I figured it would be a good idea to start standardizing on our code formatting. [PEP8](https://www.python.org/dev/peps/pep-0008/) seems widely adopted. There are 2 rules I set to ignore: really long lines (E501), and W503, which I could not make the formatter happy with anything I tried.

No code logic changes in this PR. This is purely formatting changes. Might have added 1 or 2 extra log lines.

* **Solution**: Implement PEP8 formatting. Use `pycodestyle` to check compliance
* **Implementation**: Edit code. Fix code.
* **Performed tests**: `pycodestyle src/`
* **Documentation**: Updated contributors.txt with mention of code style

* **Check list**:

[NA] I extended the Travis CI test units with the corresponding tests for this new feature (if needed).
[X] I extended the Sphinx documentation (if needed).
[NA] I extended the help section (if needed).
[NA] I changed the README file (if needed).
[NA] I created a new issue if there is further work left to be done (if needed).

**Work effort**: 8hrs

### CODE REVIEW

**Comment by utdrmac** [comment]
If we want to adopt this, should we implement a pre-hook like this? https://github.com/cbrueffer/pep8-git-hook/blob/master/pre-commit

File: src/cli/simple_client_manager.py [hunk 1]
```diff
@@ -9,14 +9,17 @@ def __init__(self, client_path, node_addr, verbose=None) -> None:
         self.client_path = client_path
         self.cmd_manager = CommandManager(verbose)
         self.node_hostname = "127.0.0.1"
-        self.node_port  =  8732
```
Comment (jdsika) [inline src/cli/simple_client_manager.py:12]: I am not happy with leaving "magic numbers" anywhere in the code. I suggest to put the standard port into a global constant and only use the constant in the code.

**Comment by jdsika** [comment]
That seems like quite some work :D

**Comment by utdrmac** [comment]
> That seems like quite some work :D

:D It was just some tedious editing. Run pycodestyle, edit files to fix issue, rinse-repeat until no more issues.

**Comment by amzid** [comment]
I suggest merging the other pending PRs that are not WIP, then taking the new master branch, running the formatting over it and resolving the merge conflicts.

**Comment by utdrmac** [comment]
The were 3, single-line conflicts dealing with comments/docs. Nothing logic changing. Easy resolve.

**Review by amzid — APPROVED** [review]

**Pull request merged**